### PR TITLE
Platform Business App - Travel request and OFF-Board Automation -Chaudhryma

### PR DESCRIPTION
**Pull request by Mariachaudhry**

### CODE REVIEW

**Review by Mariachaudhry — COMMENTED** [review]
Reviewed by me!

File: src/test/java/ServiceNow/PlatformBusinessApps/OFFBOARD/Features/CBIIT OffBoarding Form.feature
Comment (hamidnesar) [inline src/test/java/ServiceNow/PlatformBusinessApps/OFFBOARD/Features/CBIIT OffBoarding Form.feature:1]: Are all these scenarios part of the daily scheduled smoke jobs? I see that you have tagged them all @Smoke

File: src/test/java/ServiceNow/PlatformBusinessApps/OFFBOARD/Steps/CBIIT_OFFBOARD_Form_Steps.java [hunk 1]
```diff
@@ -125,4 +133,123 @@ public void i_fill_out_an_offboarding_form_for_under_the_field(String transfer,
     public void should_not_be_required_for_transfer_request(String hardwareReturnTicketNumbers) {
         OFFBOARD_StepsImpl.should_not_be_required_for_transfer_request(hardwareReturnTicketNumbers);
     }
+
+    @When("I select {string} under the {string} field")
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/PlatformBusinessApps/OFFBOARD/Steps/CBIIT_OFFBOARD_Form_Steps.java:137]: This is missing a Javadoc

File: src/test/java/ServiceNow/PlatformBusinessApps/SNOW_TRAVEL/playwright/Features/Travel_Request_Portal_Form.feature [hunk 2]
```diff
@@ -39,4 +39,10 @@ Feature: SNOWTRAVEL Portal Form Scenarios
     Given I am an authenticated DOC CGH Travel Request User
     When I am on the "Travel Planning System" form
     Then I will not see the Header on the form "Travel Cash Advance",
-    And the following fields "Cash (Direct Deposit) Advance Requested" and "Reason for Cash Advance" under Travel Cash Advance section will be removed
\ No newline at end of file
+    And the following fields "Cash (Direct Deposit) Advance Requested" and "Reason for Cash Advance" under Travel Cash Advance section will be removed
+
+  @SNOWTRAVEL-121 @Chaudhryma @Smoke @Regression @playwright
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/PlatformBusinessApps/SNOW_TRAVEL/playwright/Features/Travel_Request_Portal_Form.feature:44]: Just making sure if this test case is going to part of the daily scheduled smoke jobs, if not, @Smoke can be removed

File: src/test/java/ServiceNow/PlatformBusinessApps/SNOW_TRAVEL/playwright/Steps/Travel_Request_Portal_Form_Steps.java [hunk 3]
```diff
@@ -111,4 +115,38 @@ public void i_will_not_see_the_header_on_the_form(String travelCashAdvance) {
...
+
+    @Given("I am not a CGH user")
+    public void i_am_not_a_cgh_user() {
+        Playwright_ServiceNow_Common_Methods.side_Door_Test_Account_Login_Impersonate("Warren Kibbe");
+    }
+
+    @When("I log in to the NCI at Your Service Portal,")
+    public void i_log_in_to_the_nci_at_your_service_portal() {
+        page.navigate(TestProperties.getNCISPUrl());
+        CucumberLogUtils.playwrightScreenshot(page);
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/PlatformBusinessApps/SNOW_TRAVEL/playwright/Steps/Travel_Request_Portal_Form_Steps.java:127]: These two steps are missing Javadocs,

**Review by hamidnesar — CHANGES_REQUESTED** [review]
Hi Maria, please review the comments. thanks!

File: src/test/java/ServiceNow/PlatformBusinessApps/SNOW_TRAVEL/playwright/Features/Travel_Request_Portal_Form.feature (hunk 2 above)
Comment (Mariachaudhry) [reply src/test/java/ServiceNow/PlatformBusinessApps/SNOW_TRAVEL/playwright/Features/Travel_Request_Portal_Form.feature:44]: That is the plan!

File: src/test/java/ServiceNow/PlatformBusinessApps/SNOW_TRAVEL/playwright/Steps/Travel_Request_Portal_Form_Steps.java (hunk 3 above)
Comment (Mariachaudhry) [reply src/test/java/ServiceNow/PlatformBusinessApps/SNOW_TRAVEL/playwright/Steps/Travel_Request_Portal_Form_Steps.java:127]: added

File: src/test/java/ServiceNow/PlatformBusinessApps/OFFBOARD/Features/CBIIT OffBoarding Form.feature
Comment (Mariachaudhry) [reply src/test/java/ServiceNow/PlatformBusinessApps/OFFBOARD/Features/CBIIT OffBoarding Form.feature:1]: I added smoke tags, once Regression job is scheduled then smoke will be removed

**Review by hamidnesar — APPROVED** [review]
Reviewed, looks good!

**Review by sarwarahmed1 — APPROVED** [review]
Looks good!

**Review by katm007 — APPROVED** [review]

**Pull request merged**